### ISSUE TITLE
[CLOSED] Display error on tracks longer than 1 hour

### ISSUE THREAD

**Issue by sandrotosi** [puddletag/puddletag#497]
<a href="https://github.com/Arnvidr"><img src="https://avatars2.githubusercontent.com/u/44213979?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [Arnvidr](https://github.com/Arnvidr)**
_Fri Jun  5 12:24:03 2020_
_Originally opened as https://github.com/sandrotosi/puddletag/issues/18_

----

Tracks that have a length above 1 hour (at least that seems to be the breaking point), the length in puddletag seems to have a calculation problem or something like that. For example, for a track that soxi reports
> Duration       : 01:13:34.87 = 194695620 samples = 331115 CDDA sectors

shows in puddletag as
> 1:13.566666666666666:34

In other words, the hour is correct, and the seconds are correct, but the minutes show a fraction.

A different example, where the seconds are also wrong, is on an extremely long almost 9 hour track, where soxi reports 08:57:46.53 and puddletag shows 8:57.75:45
